### PR TITLE
React Native documentation

### PR DESCRIPTION
**Pull request by prupipho**
Starting to see how to reorganise and update the readme to include React Native

## PR checklist

- Documentation updates to include React Native
- Which issue does this PR relate to? #1668, #1671

### CODE REVIEW

**Comment by prupipho** [comment]
We will reopen this when most of the changes are applied

File: README.md [hunk 1]
```diff
@@ -18,62 +18,49 @@ are merged together by [Core Template Studio](https://github.com/Microsoft/CoreT
...
 ## Example Scenario
 
-I need a full-stack web app that is deployed on Azure App Service Web App.
+### 1. Scenario 1
+ I need a full-stack web app that is deployed on Azure App Service Web App.
 
 <img alt="App with Azure Services" src="./docs/resources/readme-app-azure-screenshot.png" width="95%" />
 
+### 2. Scenario 2
+I want to create a react native app that works for Android and Windows.
```
Comment (sibille) [inline README.md]: We should also mention iOS

File: README.md [hunk 2]
```diff
@@ -18,62 +18,49 @@ are merged together by [Core Template Studio](https://github.com/Microsoft/CoreT
...
+To verify the integrity of the package, download the `vsix` from the Marketplace, calculate the hash of the package using the powershell command `Get-FileHash` or any other tool that allows you to calculate the packages hash, and compare both.
 
 ## Instructions To Use
 
 ### Create a new project
 
 - Open **VSCode**
 - Press `Ctrl + Shift ⇧ + P` in Windows/Linux or `Command ⌘ + Shift ⇧ + P` in Mac to open VSCode's extension launcher
-- Type/Select `Web Template Studio: Launch` and press `Enter` to launch the extension
+- Type/Select `Web Template Studio: Create Web App` for a full-stack web application or `Web Template Studio: Create React Native App` for a React Native one, and press `Enter` to launch the extension
```
Comment (sibille) [inline README.md:38]: We should mention React Native is in Preview

File: README.md (hunk 1 above)
Comment (sibille) [reply README.md]:
```suggestion
I want to create a react native app that works for Android, iOS and Windows. I want my App to have a master-detail and a settings page, that allows the users to switch the App's theme.
```

File: README.md [hunk 3]
```diff
@@ -18,62 +18,49 @@ are merged together by [Core Template Studio](https://github.com/Microsoft/CoreT
...
 
+### 2. Scenario 2
+I want to create a react native app that works for Android and Windows.
+
+<img alt="App with Azure Services" src="./docs/resources/webts-dark-theme-react-native.png" width="95%" />
+
+Android App             |  Windows App
+:-------------------------:|:-------------------------:
+<img alt="Basic React Native app for Android" src="./docs/resources/react-native-sample-android-app.png" width="550" />  |  <img alt="Basic React Native app for Windows" src="./docs/resources/react-native-sample-windows-app.png" />
+
```
Comment (sibille) [inline README.md]: Not sure if this should be here or on the description of the settings/master-detail page. 
We should include screenshots for iOS too.

Comment (prupipho) [reply README.md]: Will move this screenshots without table-format to page info

**Comment by prupipho** [comment]
Outstanding to add iOS screenshots in Settings and Master Detail page

**Review by sibille — APPROVED** [review]

**Pull request merged**